ts/session/utils/Messages.ts

```diff
@@ -1,21 +1,29 @@
 import { RawMessage } from '../types/RawMessage';
-import { ContentMessage } from '../messages/outgoing';
+import {
+  ContentMessage,
+  SyncMessage,
+  OpenGroupMessage,
+} from '../messages/outgoing';
 import { EncryptionType, PubKey } from '../types';
+import { OpenGroup } from '../types/OpenGroup';
 
 export function toRawMessage(
-  device: PubKey,
-  message: ContentMessage
+  device: PubKey | OpenGroup,
```

Comment:
open group messages shouldn't be able to be converted to `RawMessage`

Reply:
Just fixed this one up mb